### DISCUSSION TITLE
Getting 2FA SMS from random names and numbers.

### DISCUSSION THREAD

**Discussion by sehajsasan** [community/community#118401] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

I use SMS for 2 factor authentication for my github account. I am getting weird SMS with the otp,where it is nowhere mentioned that it is a github authentication SMS, and random company names are written. When I use that OTP it works. I am concerned about the security implications.

![WhatsApp Image 2024-04-09 at 1 22 21 PM](https://github.com/community/community/assets/50807883/d98faf95-ec17-4c20-bf9a-25031a190e2b)

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @sehajsasan! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by ishavgupta** [comment]
Same happens with me when using otps for other apps/websites.
Do you know why this happens?
![Uploading IMG_5624.png…]()
